exchanges/throw-on-error/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@urql/exchange-throw-on-error",
+  "version": "0.0.0",
+  "description": "An exchange for throw-on-error support in urql",
+  "sideEffects": false,
+  "homepage": "https://formidable.com/open-source/urql/docs/",
+  "bugs": "https://github.com/urql-graphql/urql/issues",
+  "license": "MIT",
+  "author": "urql GraphQL Contributors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/urql-graphql/urql.git",
+    "directory": "exchanges/throw-on-error"
+  },
+  "keywords": [
+    "urql",
+    "graphql client",
+    "graphql",
+    "exchanges",
+    "throw on error"
+  ],
+  "main": "dist/urql-exchange-throw-on-error",
+  "module": "dist/urql-exchange-throw-on-error.mjs",
+  "types": "dist/urql-exchange-throw-on-error.d.ts",
+  "source": "src/index.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/urql-exchange-throw-on-error.d.ts",
+      "import": "./dist/urql-exchange-throw-on-error.mjs",
+      "require": "./dist/urql-exchange-throw-on-error.js",
+      "source": "./src/index.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "LICENSE",
+    "CHANGELOG.md",
+    "README.md",
+    "dist/"
+  ],
+  "scripts": {
+    "test": "vitest",
+    "clean": "rimraf dist",
+    "check": "tsc --noEmit",
+    "lint": "eslint --ext=js,jsx,ts,tsx .",
+    "build": "rollup -c ../../scripts/rollup/config.mjs",
+    "prepare": "node ../../scripts/prepare/index.js",
+    "prepublishOnly": "run-s clean build"
+  },
+  "devDependencies": {
+    "@urql/core": "workspace:*",
+    "graphql": "^16.0.0",
+    "vitest": "^0.30.1"
```

Comment:
Can we make this the latest version like our root

Reply:
Done! Btw, I have just copy-pasted it from the other exchanges. Maybe we should adapt pnpm catalogs to get rid of these kinds of issues?

Reply:
Looks like the newer Vitest versions are not working due to [this](https://github.com/urql-graphql/urql/blob/6408474fdbb841900caba70f7c6ff600a12768cf/package.json#L55)? Should I rollback for now?